src/frontend/main/MapView.tsx

```diff
@@ -171,12 +176,11 @@ const MapView = (props: MapViewProps) => {
   const currentPath = props.location!.pathname;
   const isEmbed = currentPath === '/map-embed';
   const topPartHeight = isEmbed ? NAVHEIGHT : HEADERHEIGHT + NAVHEIGHT;
-  const { isShowing, toggleModal } = useModal();
   const [showMapInfo, setShowMapInfo] = useState(true);
   const [selectedFilter, setSelectedFilter] = useState<FilterKey>(FILTERS.corona_suspicion_yes.id as FilterKey);
   const [mapHeight, setMapHeight] = useState(window.innerHeight - topPartHeight - 10);
-  const [activeCityData, setActiveCityData] = useState({});
   const data = props.responseData.data;
+  const [activeView, setActiveView] = useState<'MAP' | 'TABLE'>('MAP');
```

Comment:
This is nice and tidy! :ok_hand:

Reply:
Thanks, I used your solution for the data state in App.tsx as an example!

Reply:
Gah, it's always awkward when you give a compliment and end up complimenting yourself...